### ISSUE TITLE
Update number of days to complete disability-related claims for January 2023

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#53020]
## Issue Description
We need to update the number of days it takes to complete disability related claims. Sitewide content needs to email Tanekwa Bournes on the first of the month to get the number from the previous month and input into Drupal.

This needs to be done on the following pages:
https://www.va.gov/disability/after-you-file-claim/
https://www.va.gov/disability/how-to-file-claim/

## Tasks
- [x] Email Tanekwa for most recent number (Randi)
- [x] Input number and new month into both Drupal pages (Aliyah)
- [x] Review and publish (Megan)

## Acceptance Criteria
- [x] Most current number of days is on live site.

**Comment by RLHecht** [comment]
@aliyahblackmore The number for January is 103.5. Thanks!

**Comment by aliyahblackmore** [comment]
Hi @megzehn these are ready for you! 

**After you file a claim**
**Preview link:** http://preview-prod.vfs.va.gov/preview?nodeId=870
Note: I made one adjustment here to the link format. I just confirmed with Randi that we should leave the "Track the status of your claim" link as a default link.

**How to file a claim**
**Preview link:** http://preview-prod.vfs.va.gov/preview?nodeId=905 
Note: In the bulleted list, there were some changes made in lead up to translation, but it seems like there was a bullet point removed and one other adjustment in Katherine's work on the YouTube link placement. Just want to confirm these adjustments!
